hathor/verification/verification_service.py

```diff
@@ -274,7 +275,6 @@ def _verify_without_storage_block(self, block: Block) -> None:
         self._verify_without_storage_base_block(block)
 
     def _verify_without_storage_merge_mined_block(self, block: MergeMinedBlock) -> None:
-        self.verifiers.merge_mined_block.verify_aux_pow(block)
```

Comment:
Why can't we run this verification without storage? This change seems completely unrelated to the other refactors in this PR.

Reply:
As explained in slack, this was wrong as this verification does in fact use the storage.

Reply:
Got it. Thanks!